src/Couchbase.Lite.Shared/API/Document/IDictionaryObject.cs

```diff
@@ -53,15 +53,13 @@ public interface IDictionaryObject : IDictionaryFragment, IEnumerable<KeyValuePa
         /// </summary>
         /// <param name="key">The key to check for</param>
         /// <returns><c>true</c> if the dictionary contains the key, else <c>false</c></returns>
-        [ContractAnnotation("null => halt")]
```

Comment:
Here are some places that have the contract, but are missing NotNull.  They should get NotNull to become consistent with the rest of the PR.